packages/neutrino-preset-web/polyfills.js

```diff
@@ -1 +0,0 @@
-import 'babel-polyfill';
```

Comment:
The `babel-polyfill` package is now unused too and can be removed from package.json

Reply:
Definitely, good catch.

Reply:
I've opened #339 to remove the unused deps